gcloud/streaming/exceptions.py

```diff
@@ -138,7 +133,7 @@ class BadStatusCodeError(HttpError):
     """The request completed but returned a bad status code."""
 
 
-class NotYetImplementedError(GeneratedClientError):
+class NotYetImplementedError(Error):
```

Comment:
`BufferedStream.read()` raises it, but only for specific cases (the passed `size` is None, or less than zero).  I'd be fine with converting it to a ValueError (along with most of the custom errors raised by this package).